services/drupal/web/sites/default/settings.php

```diff
@@ -915,6 +915,11 @@
 // Ensure we force the site to use the "include" method of shielding pages
 $config['shield.settings']['method'] = 1;
 
+// Set the required configuration for epa_cloudwatch.
+$config['epa_cloudwatch']['region'] = 'us-east-1';
+$config['epa_cloudwatch']['version'] = '2014-03-28';
+$config['epa_cloudwatch']['log_group'] = '/webcms/drupal';
```

Comment:
I have two comments:

1. In order to be environment-agnostic, let's read the region, log group, and log stream from the environment using `getenv()`. We already have `WEBCMS_S3_REGION` for the region, and it should be pretty easy to just use `WEBCMS_LOG_GROUP` to name the log group to write to. (The group name varies per-environment, so we can't use a fixed name.)
2. Since we're expecting a certain API version within our calls to the AWS API, does it make sense to make the API version configurable here? I would argue no, since this value is pretty tightly coupled to `epa_cloudwatch`'s internals.

Reply:
I made the following updates:
* Updated the module to read `WEBCMS_S3_REGION` for the region
* Created a new environment variable for `WEBCMS_LOG_GROUP`, adding it to terraform to read from parameter store and set added it to `.env.example`. I set the example value to `/webcms-local` to match the naming scheme already in place in Cloudwatch
* Moved the version definition into the module